python/servo/testing_commands.py

```diff
@@ -164,16 +164,20 @@ def test_perf(self, base: str | None = None, date: str | None = None, submit: bo
     )
     @CommandArgument("--code-coverage", default=False, action="store_true", help="Run in code coverage mode")
     @CommandArgument("--llvm-cov-option", default=None, action="append", help="Additional options for llvm-cov")
+    @CommandArgument("--nextest-profile", default=None, help="Specify the Nextest profile to use")
+    @CommandArgument("params", nargs="...", help="Command-line arguments to be passed through to Cargo nextest")
```

Comment:
`params` is not strictly related to this PR, but it allows passing through parameters to nextest, like `--stress-count` to rerun tests x amount of time, which is quite useful when trying to reproduce flaky tests.